plugins/README.md

```diff
@@ -0,0 +1,29 @@
+# LOOP Plugins
+
+:warning: Experimental :warning:
+
+This directory supports Local-Out-Of-Process (LOOP) Plugins, an alternative node runtime where some systems execute in 
+separate processes, plug in via [github.com/hashicorp/go-plugin](https://github.com/hashicorp/go-plugin), and 
+communicate via [GRPC](https://grpc.io).
+
+There are currently two kinds of plugins, and one implementation of each: a Solana Relayer plugin, and a Median Reporting
+plugin. The [cmd](cmd) directory contains their `package main`s for now. These can be built via `make install-solana` and 
+`make install-median`.
+
+## How to use
+
+[chainlink.Dockerfile](chainlink.Dockerfile) extends the regular [core/chainlink.Dockerfile](../core/chainlink.Dockerfile)
+to include the plugin binaries, and enables support by setting `CL_SOLANA_CMD` and `CL_MEDIAN_CMD`. Either plugin can be
+disabled by un-setting the environment variable. Images built from this file can otherwise be used normally, provided that the pre-requisites have been met.
```

Comment:
>Images built from this file can otherwise be used normally, provided that the pre-requisites have been met.

I don't understand this sentence. what does `normally` mean here? without plugins? what are the requisites?

It reads as though you can use this image to run the node as you always have but some pre-requisites may be different for this image than others.

Reply:
'Normally' as in the normal image built from the regular Dockerfile. The point is that the UX does not change other than those two env vars.